crowd/actions/add_user_to_group.py

```diff
@@ -0,0 +1,15 @@
+from lib.base import CrowdBaseAction
+
+class CrowdAddUserToGroupAction(CrowdBaseAction):
+	def run(self, username, groupname):
+		# Return True if the user exists in the Crowd application.
+		if not self.status:
+			msg = 'Application authentication Failed.'
+			return(False, msg)
+			
+		result = self.crowd.add_user_to_group(username, groupname)
+		
+		if result:
```

Comment:
You could perhaps simplify this and collapse it into a single line of code by doing ``return (bool(result), result)``, but it's not a blocker.